packages/lesswrong/components/posts/LWPostsItem.tsx

```diff
@@ -428,139 +432,142 @@ const LWPostsItem = ({classes, ...props}: PostsList2Props) => {
           })}
         >
-          <PostsItemTooltipWrapper
-            post={post}
-            placement={tooltipPlacement}
-            className={classNames(
-              classes.postsItem,
-              classes.withGrayHover, {
-                [classes.dense]: dense,
-                [classes.withRelevanceVoting]: !!tagRel,
-                [classes.hasSequenceImage]: !!resumeReading,
-              }
-            )}
-          >
-            {tagRel && <Components.PostsItemTagRelevance tagRel={tagRel} />}
-            {showKarma && <PostsItem2MetaInfo className={classNames(
-              classes.karma, {
-                [classes.karmaPredictedReviewWinner]: !!marketLink
-              })}>
-              {post.isEvent
-                ? <AddToCalendarButton post={post} />
-                : <KarmaDisplay document={post} linkItem={marketLink}/>
+          <div {...eventHandlers}>
+            <PostsItemTooltipWrapper
+              post={post}
+              placement={tooltipPlacement}
+              className={classNames(
+                classes.postsItem,
+                classes.withGrayHover, {
+                  [classes.dense]: dense,
+                  [classes.withRelevanceVoting]: !!tagRel,
+                  [classes.hasSequenceImage]: !!resumeReading,
+                }
+              )}
+            >
+              {tagRel && <Components.PostsItemTagRelevance tagRel={tagRel} />}
+              {showKarma && <PostsItem2MetaInfo className={classNames(
+                classes.karma, {
+                  [classes.karmaPredictedReviewWinner]: !!marketLink
+                })}>
+                {post.isEvent
+                  ? <AddToCalendarButton post={post} />
+                  : <KarmaDisplay document={post} linkItem={marketLink}/>
+                }
+              </PostsItem2MetaInfo>}
+
+              <span className={classNames(classes.title, {[classes.hasSmallSubtitle]: !!resumeReading})}>
+                <AnalyticsTracker
+                    eventType={"postItem"}
+                    eventProps={{mountedPostId: post._id, mountedPostScore: post.score, mountedPostBaseScore: post.baseScore}}
+                    captureOnMount={(eventData) => eventData.capturePostItemOnMount}
+                    captureOnClick={false}
+                >
+                  <PostsTitle
+                    postLink={postLink}
+                    post={post}
+                    read={isRead && !showReadCheckbox} // readCheckbox and post-title read-status don't aesthetically match
+                    sticky={sticky}
+                    showDraftTag={showDraftTag}
+                    {...(showPersonalIcon ? {showPersonalIcon} : {})}
+                    curatedIconLeft={curatedIconLeft}
+                    strikethroughTitle={strikethroughTitle}
+                    showRecommendationIcon={isRecommendation && showRecommendationIcon}
+                    postItemHovered={hover}
```

Comment:
Is there any real change in this file except wrapping the whole thing in a div with the hover event handlers, and passing in the hover here, with the rest just being indentation?